js/components/portfolio/OperationsComponent.js

```diff
@@ -0,0 +1,123 @@
+/**
+ * CITTADINANZA DIGITALE
+ * io.italia.it
+ *
+ * @flow
+ */
+
+import * as React from 'react'
+import I18n from '../../i18n'
+import {
+  Body,
+  Grid,
+  Icon,
+  Left,
+  List,
+  ListItem,
+  Right,
+  Row,
+  Text
+} from 'native-base'
+import type { NavigationScreenProp, NavigationState } from 'react-navigation'
+import type { CreditCard, Operation } from '../../lib/portfolio/types'
+import PortfolioAPI from '../../lib/portfolio/portfolio-api'
+import { withNavigation } from 'react-navigation'
+import ROUTES from '../../navigation/routes'
+
+type Props = {
+  parent: string,
+  navigation: NavigationScreenProp<NavigationState>,
+  operations: $ReadOnlyArray<Operation>
+}
+
+type State = {
+  data: $ReadOnlyArray<Operation>
+}
+
+/**
+ * Operations' List component
+ */
+class OperationsList extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props)
+    this.state = { data: Array.from(props.operations) }
+  }
+
+  renderDate(operation: Operation): React.Node {
+    const datetime: string = `${operation.date} - ${operation.time}`
+    if (operation.isNew) {
+      return (
+        <Row>
+          <Icon
+            type="FontAwesome"
+            name="circle"
+            active
+            style={{ marginTop: 6, fontSize: 10, color: '#0066CC' }}
```

Comment:
please do not spread magic numbers for sizes and/or colors, use constants of array of constants instead.

if you have a component that may repeat in the UI please make a stateless styled component instead so it's reusable.

@fpersico do we have a guidelines or some resource we can point them out ?